.github/workflows/ci.yml

```diff
@@ -81,6 +81,10 @@ jobs:
             os: ubuntu-20.04  # CPython 3.9.2 is not available for ubuntu-22.04.
             experimental: false
             nox-session: test-3.9
+          - python-version: "3.11"
```

Comment:
More a wondering, don't feel the need to implement this in this PR: are we going to need to test multiple browsers? How much does emscripten support differ across browsers, do you have insights into this @joemarshall?

Reply:
It should test on firefox, but there's an outstanding issue in pytest-pyodide that means you can't turn off certificate verification in firefox easily.

I started with 3.11 because the stable version of pyodide I'm testing against uses it, and I find everything is easier if you don't mix and match host and pyodide versions of python too much.